styles/scss/_articles.scss

```diff
@@ -4,15 +4,22 @@
   .section__heading--articles {
     text-align: center;
     font-weight: 500;
-    @media screen and (min-width: 768px) {
-      text-align: left;
+    @media screen and (min-width: 768px){
       padding-bottom: 30px;
     }
   }
+  .box--articles{
+    display: grid;
+    gap: 10px;
+    @media screen and (min-width: 768px){
```

Comment:
@avtill  Tutaj w zasadzie to pierwsze media query nie jest potrzebne, powinno zostać samo `grid-template-columns: 1fr 1fr 1fr 1fr; `.

Reply:
@avtill wybacz, źle spojrzałam - myślałam, że masz nadmiarowe @media, ale Ty miałaś w jednych _columns_ zawarte, a  w drugich _rows_, więc to nie ten przypadek :D
Ale tak teraz zaczęłam myśleć nad tym - zbudowałam sobie Twojego brancha i chyba nie do końca mi się to zgrywa. Masz tu podział na 4 kolumny dla przedziału już od min-width:768px w górę - więc dla rozdzielczości od 768px do ok. 1100px te 4 kolumny wychodzą poza stronę.
![image](https://user-images.githubusercontent.com/22479664/236324193-b691de90-47ee-4905-b666-9c22cf230433.png)

Dodatkowo ostatni item ucieka trochę w górę:
![image](https://user-images.githubusercontent.com/22479664/236325126-e95c79ad-1f8a-4d3d-9a8a-ae63502d879e.png)

Druga rzecz: 
Na rozdziałkach poniżej 768px grafiki się rozciągają
![image](https://user-images.githubusercontent.com/22479664/236325273-e7cd946c-2e13-48cb-baed-8c1a93409ee4.png)

I trzecia sprawa:
Artykuły mają dorzucony padding 50px, który wydaje mi się zbędny - wg projektu wszystko jest zamknięte w jednym kontenerze _max-width: 1140px_ z _padding: 0 15px;_ (na mobilkach _padding: 0 25px_) i tak też mamy w kodzie - _klasa body__container--main_ (widzę teraz też, że muszę poprawić tam jedną rzecz, która powoduje te wielkie przestrzenie między wierszami na mobilkach, ale to poprawię już w moim tickecie).